### DISCUSSION TITLE
Struggling with Tailscale to remote access

### DISCUSSION THREAD

**Discussion by GingerTom88** [photoprism/photoprism#4022] [General]
I'm proper struggling to use Tailscale to access PhotoPrism remotely, which sucks as I want to use PhotoPrism to allow family to access my photos.

I am running PhotoPrism on my NAS with TrueNAS Scale Bluefin.

I've followed this guide https://docs.photoprism.app/getting-started/vpn/tailscale/ and have installed Tailscale on my TrueNAS. I can see my TrueNAS showing up in my Tailscale admin panel, but when I use the Tailscale IP address it doesn't connect. 

Can anyone suggest anything to help?

**Comment by lastzero** [comment]
Please note that this guide was contributed by our community, so we have not thoroughly reviewed or tested it. Also, Tailscale is a young company that moves fast. Certain details or settings may therefore have changed since it was written. If you have any suggestions for improvement, you can let us know by clicking on the Edit button in the upper right corner. Thank you very much!
